tests/cli/Cli/Task/SetEventsManagerCest.php

```diff
@@ -26,6 +28,9 @@ public function cliTaskSetEventsManager(CliTester $I)
     {
         $I->wantToTest('Cli\Task - setEventsManager()');
 
-        $I->skipTest('Need implementation');
+        $eventsManager = new Manager();
+        $task = new Task();
+        $task->setEventsManager($eventsManager);
```

Comment:
The same as above

Reply:
I think you can remove this one in favour of the other one.